### ISSUE TITLE
Not getting results

### ISSUE THREAD

**Issue by mojoyup** [samsface/godot-steam-api#14]
Hey :) I followed the youtube video, found that things were a bit different and then followed the instructions here also and made sure EVERYTHING was properly set up. txt file, sdk source path, appid, singleton, enabled, etc etc. And I am getting no achievement from it. REstarted Steam, made sure the overlay showed me online, etc etc. Any ideas?

**Comment by mojoyup** [comment]
Steam does load on the example and I can Shift Tab to it just fine, so the initialize is working and logged in.

**Comment by samsface** [comment]
You can test without uploading to steam (just the overlay functions won't work).

Can you share the code you're using to unlock the achievement?
Try run locally and see if any error is printed to console, if there is pasted it here.

Also did you publish your changes in steam works? Every-time you change achievements you need to publish them (same button to publish your store page).

**Comment by mojoyup** [comment]
I am using just the code that came with example.gd. I left everything default except for what I needed to change (appid, sdk, etc). 

I ran it from godot scene(F6) as well as exported, both no show. There are no errors during the run. 

I added an achievement in steamworks and did publish everything (though it may not work on my own achievements yet because I am still in "coming soon" stage. But I would think the default like you use in the video would work right? Or was that just to show the achievement that you had setup already in steamworks? 

Just hoping Im not missing anything. I also added the text appid file so I could run without uploading and no go. I put it in the exe location and also have the one that is added by the code in the godot executable location. 

Thanks for helping. It may be something simple I am missing but I am very positive I did the steps as they were simple to follow.

**Comment by mojoyup** [comment]
So, when I use the code from github on the instructions, I do get an error though: Invalid call. Nonexistent function 'activate_game_overlay_to_web_page' in base 'Reference ()'.

Otherwise, if I just do the:
Steam.clear_achievement("gator_god")
Steam.set_achievement("gator_god")

like in the video, I get no errors

Hope that helps!

**Comment by samsface** [comment]
If there's no errors then it's probably a configuration issue :thinking: 

I'd double check you're using the right names for the achievement, you set the correct permissions (so can be set by client) and publish the changes in steam works.

Another thing someone else saw was checking and unchecking the "Disable Steam Integration" fixed their issues.

**Comment by mojoyup** [comment]
Ok thank you very much. I will look into making sure everything is good. So, basically, I cant use the default achievement because it is not part of my Steamworks (it's something you had set up); I have to use my own achievement even when testing, correct?

Therefore, the gator_god achievement isnt going to work for us, just for you because it was setup on your Steamworks. Is that correct? Appreciate your time!

**Comment by samsface** [comment]
Yeah I setup gator_god in my steamworks page. You need to setup your own.

**Comment by mojoyup** [comment]
I think it may be then just that my app is still in coming soon. Im sure itll work over time :) Thank you for being patient with me and helping me understand it more.
